.docker/Dockerfile

```
@@ -0,0 +1,50 @@
+FROM ubuntu:bionic AS build
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends checkinstall debhelper g++ gcc git glib2.0 intltool libxml2-dev meson pkg-config unzip wget && \
+    rm -rf /var/lib/apt/lists/*
```

```suggestion
ARG DEBIAN_FRONTEND=noninteractive

RUN apt-get update \
 && apt-get install -y --no-install-recommends \
      checkinstall \
      debhelper \
      g++ \
      gcc \
      git \
      glib2.0 \
      intltool \
      libxml2-dev \
      meson \
      pkg-config \
      unzip \
      wget \
 && rm -rf /var/lib/apt/lists/*
```

Using `ARG` prevents `DEBIAN_FRONTEND` from being set to `noninteractive` in derived `Dockerfile`s while still setting it for everything in this particular `Dockerfile`.

And I would put each package on its own line, as it's eas(y)(ier) to see which packages get installed (and to add/remove something).

---

Thanks for the input, I've included the changes in the commit I will push shortly.